app/controllers/tasks.js

```diff
@@ -18,6 +19,11 @@ export default class TasksController extends Controller {
   @tracked allTasks = this.model;
   @tracked isLoading = false;
   @tracked userSelectedTask = this.DEFAULT_TASK_TYPE;
+  @tracked showModal = false;
+  @tracked tempTaskId = ''; // this Id will be used to update task which are completed 100%
+  @tracked message = ''; // this is required in the modal
+  @tracked buttonRequired = false; // this is required in the modal
```

Comment:
Can we name it better?
What it is for??

Reply:
which variable are you talking about? there are four variables here

Reply:
hey rohan the buttonRequired is used to tell if we require a button for the modal or not, I guess the naming is correct